.github/workflows/docker-build-run.yml

```diff
@@ -1,6 +1,6 @@
 name: Build Docker image and test container
 
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
We still want this to build on master though? Not sure what's the best way to avoid redundant CI jobs.

Reply:
Yes, but sometimes CI can pass in PRs and fail on master. I think what we want is something like, https://help.github.com/en/actions/reference/events-that-trigger-workflows#example-using-multiple-events-with-activity-types-or-configuration

Let's merge as is, I'll open a separate PR.